trade_remedies_api/core/management/commands/deactivate_users.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from core.models import User
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """
+    Management command to deactivate users.
+    """
+
+    help = "Command to deactivate users."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--emails", type=str, help="CSV list of user emails", required=True
+        )
+
+    def handle(self, *args, **options):
+        logging.info("Deactivating users")
+
+        user_email_list = options["emails"].split(",")
+
+        qs = User.objects.filter(email__in=user_email_list, is_active=True)
```

Comment:
I think we basically need to inverse this, i.e. only keep the users that we have passed as an argument and de-activate the rest of them.

Seems quite intense but otherwise we need to do a separate query to get the users which do match the set criteria and we may miss some.

```suggestion
        qs = User.objects.exclude(email__in=user_email_list, is_active=True)
```